### ISSUE TITLE
potential gameplay exploitation issue

### ISSUE THREAD

**Issue by jxsl13** [jxsl13/zcatch#103]
Full server,
one winning player with 13 caught players
one ingame player
another ingame player

one of the ingame players is caught and leaves the game.

While the second ingame player is alive, any of the spectating (not caught) players may join the game, leading to  the winning player having to catch 63 players in the worst case in a row.

Easy workaround: 
One enemy player left: do not allow joining

Might make the game boring in some edge cases where only a few players are currently ingame.

An additional condition might be to look at the currently winning player (number of caught players + number of caught left players - released players(not sure if necessary)) and if that number is 15, you block players from joining.
Basically being the join condition whenever someone wants to join.

**Comment by jxsl13** [comment]
release v0.3.4 to be fixing this

**Comment by jxsl13** [comment]
pain in the butt, keeping this as is...
reverted.
